drafts/use-cases/3.obtaining-events.md

```diff
@@ -0,0 +1,110 @@
+## Entry point
+
+### Story
+As an application user
+I want to display all my events
+So I can find out what awaits me in the nearest future.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api/events");
+for (var member of data.members) {
+    // do something with the _member_, i.e. display it
+}
+```
+
+### Details
+My application used a discovered either from the entry-point or the API documentation URL to my events.
+It sends a GET request to that URL and fetches the events:
+
+```
+GET /api/events
+
+HTTP 200 OK
+```
+```javascript
+{
+    "@id": "/api/events",
+    "totalItems": 1,
+    "members": [
+        {
+            "@id": "/api/events/1",
+            "eventName": "Event 1",
+            "eventDescription": "Some event 1",
+            "startDate": "2017-04-19",
+            "endDate": "2017-04-19"
+        }
+    ]
+}
+```
+
+It is possible now to display one event in the calendar.
+
+### Considerations
+
+#### Some issues
+Unfortunately, my application is not fully aware of what will come.
+It knows only that a collection of items will be returned, but the type of these items is not disclosed.
+Also there is no hint before fetching whether the collection is complete or partial.
+Hydra Core Vocabulary allows partial collections, 
+but it's unclear how to denote that fact before fetching the collection.
+This may lead to situation when an application can be "surprised"
+either by receiving a partial collection forcing it to make additional calls to fill the view 
```

Comment:
Applications should always be prepared for this; in fact, they will not see it because the library will hide this. (Successfully implemented this pattern with the LDF query client; results are just streams—the same whether there's 1 or 5,000 pages.)